### DISCUSSION TITLE
GGUF  can no longer use GPU?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [oobabooga/text-generation-webui#5957] [Q&A]
I'm not sure if it was something did (been doing way too much at the same time lately and attention and memory haven't been firing on all cylinders lately), or if it was some update to OTGW, or some driver thing or what, but I noticed that GGUF files no longer seem to be using any VRAM at all, and there isn't even the "Use Tensor Cores", or whatever was called, checkbox when loading GGUF files anymore. Does anyone know what could be going on?

ps: CUDA on the base system seems to still be working, Blender sees it just fine and renders with no noticeable artifacts, and GPTQ and AWQ models seem to still use the GPU.

pps: This is on Linux, and I'm starting OTGW as have been for a long while, `conda activate oobabooga` followed by `./start_linux.sh`

**Comment by michusx** [comment]
I've noticed similar behawior after recent git pull and pip --update. I've checked that installed versions of all llama_cpp_python libraries didn't updated to 0.2.64 despite 'requirements.txt'.
I've updated manually with pip all 3 llama_cpp_python libraries from 'https://github.com/abetlen/llama-cpp-python/releases/' repository and it is working with GPU now...

**Reply by TiagoTiago** [reply]
There are later versions there, does compatibility break above 0.2.64?

**Reply by michusx** [reply]
Hard to say, I've reinstalled to 0.2.64 as requirements says.

**Reply by TiagoTiago** [reply]
Hm, I've just noticed something, if I understand it right seems the conda env I had been using, perhaps is being ignored by the start script and it's using it's own; so all the time trying to setup things on the original env was for nothing. The stupid thing isn't even named after Ooba or something related to "text-generation-webui", it's just generic-ass "conda". Actually, looks like it's two, there is also the equally generic "env". I'm not even sure which it's actually using...

Gotta try installing it on each of them see which one makes it work...

It can't be healthy for the system to have one app hijack a generic name like that... I wonder what might happen if another app also goes for one of those generic names...

**Reply by TiagoTiago** [reply]
Ok, reading up one somethings I'm starting to suspect I might've gotten rusty and forgot some details; gonna see what happens if I delete those two weird envs, and follow the manual Conda instalation procedure again.

**Reply by TiagoTiago** [reply]
Ok, yep, that start_linux.sh launcher was definetely switching to those one of other envs, doesn't work at all without them there anymore...

**Reply by TiagoTiago** [reply]
Way too much back and forth with deleting and undeleting files and things that were supposed to be there not being found by the app and then they are and then not once more, and I'm not quite sure how I fixed it. Kinda looks like maybe things do work with the latest version of the package though.

... I need sleep -___-

**Reply by TiagoTiago** [reply]
Ugh, not sure what changed. Spent a some time not using it, now the problem is back.... Driver update mseed something in mean time perhaps? Gonna try updating ooba and stuff see if anything changes...

Also, ugh, something changed in the meantime and the git merge on updating it triggering vim instead of whatever editor it was using before; the previous one had the important keyboard shortcuts visible at the bottom. I know some people are religious about this shit, but my brain has already too many holes, running out of neurons to memorize new obscure shortcuts.....

**Reply by TiagoTiago** [reply]
Yeah, updating seems to have fixed the GPU usage
